code/modules/mob/living/simple_animal/hostile/retaliate/cluwne.dm

```diff
@@ -164,7 +164,7 @@
 	icon_dead = null
 	response_help = "honks the"
 	speak = list("Honk!")
-	speak_emote = list("sqeaks")
+	speak_emote = list("squeaks")
```

Comment:
🤔

Reply:
fixing others past typos. wtf is a comdom

Reply:
sqeak

Reply:
this is fine